ci/jobs/README.md

```diff
@@ -75,3 +75,70 @@ Once you have the UUID of interest, add it into your job template. Each job temp
 or more private credentials to be installed in the `ssh-agent` using the `ssh-agent-credentials`
 directive. See the [`ssh-agent-credentials docs`](http://docs.openstack.org/infra/jenkins-job-builder/wrappers.html#wrappers.ssh-agent-credentials)
 for examples.
+
+Example Config
+--------------
+
+Create the config file.
+
+```commandline
+mkdir ~/.config/jenkins_jobs/
+vi ~/.config/jenkins_jobs/jenkins_jobs.ini
+```
+
+Here is an example config:
+```ini
+[jenkins]
+user=bbouters
+password=5cxr6230dptzp6l7oqga03chupnqa8f5
+url=https://pulp-jenkins.rhev-ci-vms.eng.rdu2.redhat.com
```

Comment:
Is this an internal-only URL? If so, would a URL like pulp-jenkins.example.com be better?